### ISSUE TITLE
Semantics

### ISSUE THREAD

**Issue by smeragoel** [Quansight-Labs/czi-scientific-python-mgmt#60]
## 🌐 CSS Content
Meaningful content must not be implemented using only CSS **:before** or **:after** - [WCAG 1.3.1](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships)

- [x] 	With CSS Styles disabled, the Example Gallery and "Supports PyData and Jupyter" links are not accessible.
Before:
![image](https://github.com/Quansight-Labs/czi-scientific-python-mgmt/assets/98317216/21616cdb-25f6-4ab2-a035-ca889cbcffb9)
After disabling CSS Styles:
![image](https://github.com/Quansight-Labs/czi-scientific-python-mgmt/assets/98317216/5c931c46-9148-460f-82df-dc2e67e72d3d)
![image](https://github.com/Quansight-Labs/czi-scientific-python-mgmt/assets/98317216/d0aecb93-d8ac-4b3f-8ebf-557e708b03a6)

- [ ] When activating the visual helper for the accessibility test, unexpected "lines" appear on the screen. 
![image](https://github.com/Quansight-Labs/czi-scientific-python-mgmt/assets/98317216/051d709a-5f00-4c38-bef8-25d3eb4cf467)
![image](https://github.com/Quansight-Labs/czi-scientific-python-mgmt/assets/98317216/ccc54347-79be-4ed8-b572-9a2abdff26bf)

**Comment by gabalafou** [comment]
First issue is fixed, see https://github.com/pydata/pydata-sphinx-theme/pull/1549#discussion_r1391494346

**Comment by gabalafou** [comment]
As for the second issue, the red lines are highlighting all of the pseudo-elements for the that admonition, of which there are three:

1. `.admonition::after { content: "" }` - I'm not exactly sure what purpose this pseudo-element serves, it's probably some kind of layout fix, but as its content is an empty string, it contains no meaningful content, so it's not an issue.
2. `.admonition-title::before { content: "" }` - This puts a green bar behind the title, again no content, so not an issue
3. `.admonition-title::after { content: var(--pst-icon-admonition-seealso) }` - This is the share icon. It is set to `--pst-icon-share: "\f064";`. It is purely decorative. The meaning of the icon is conveyed by the admonition title text, which is "See also." So this one is also not an issue. The pseudo-element content appears in the accessibility tree, which is probably less than ideal, but it was ignored by VoiceOver when I tested it. So I don't think this is an issue.

**Comment by gabalafou** [comment]
Given my previous comment, I going to check off the second checkbox and close this issue.
